src/Razor/src/Microsoft.AspNetCore.Razor.Language/Legacy/TagHelperBlockRewriter.cs

```diff
@@ -40,7 +40,7 @@ internal static class TagHelperBlockRewriter
 
             foreach (var descriptor in bindingResult.Descriptors)
             {
-                var boundRules = bindingResult.GetBoundRules(descriptor);
```

Comment:
It was really wierd to me that the mappings weren't exposed as a property. This cause alex and I miss that this was already present when we were designing this feature 😢 

Taylor if you prefer it the way it is, then I will revert this part of the change.

Reply:
Don't care either way 😄